src/test/app/Path_test.cpp

```diff
@@ -171,6 +171,131 @@ rpf(jtx::Account const& src, jtx::Account const& dst, std::uint32_t num_src)
     return jv;
 }
 
+// test helper
+bool checkArraySize(Json::Value const& val, unsigned int size)
+{
+    return val.isArray() &&
+           val.size() == size;
+}
+
+using verify_paths =
+    std::vector<std::vector<std::pair<ripple::Issue, jtx::Account>>>;
+
+// compares expected paths (pairs of Issue/Account) with
+// json output from path_find
+bool
+check_paths(
+    Json::Value const& jvr,
+    std::string & err,
+    jtx::PrettyAmount const& dst_amt,
+    boost::optional<jtx::PrettyAmount> const& src_amount = boost::none,
+    boost::optional<verify_paths> const& paths = boost::none)
+{
+    if(jvr[jss::destination_amount] != dst_amt.value().getJson(0))
+    {
+        err = "destination amount mismatch";
+        return false;
+    }
+
+    if(paths)
+    {
+        if(! checkArraySize(jvr[jss::alternatives], 1))
```

Comment:
General path finding question: Why is there always only 1 alternative in these tests?